ogc/src/main/scala/geotrellis/server/ogc/OgcSource.scala

```diff
@@ -51,6 +51,7 @@ trait OgcSource {
   def metadata: RasterMetadata
   def attributes: Map[String, String]
   def time: Option[ZonedDateTime]
+  def resampleMethod: ResampleMethod
```

Comment:
Is this part of the OGC definitions for "what an OGC layer is capable of" and does this attribute meet the requirements for inclusion defined in the docstring? Just double checking that this belongs as part of this trait so that we don't start to hang attributes off of OgcSource that make it less composable.

Reply:
Yes, I think it is a 'what method for all resampling operation this raster source would use'; checkout also #229

Reply:
👍